learn/generation/rag-for-hybrid/rag-for-hybrid-search.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.       dimension = 384,  # dimensionality of our vectors
</pre><p>might want to clarify it is of the dense vectors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
<p>Oh, nice! Yes I'll do that</p>